packages/cardano-services/src/Program/ProgramOptionDescriptions.ts

```diff
@@ -1,9 +1,11 @@
-export enum ProgramOptionDescriptions {
+import { CommonOptionDescriptions } from '../ProgramsCommon';
+
+enum HttpServerOptionDescriptions {
   ApiUrl = 'API URL',
   DbConnection = 'DB Connection',
```

Comment:
Isn't `DbConnection` also a "common" option?

Reply:
AFAIK no, or better, at least it's not common between start-server and start-worker